plugins/mobile-plugin/src/main/java/ru/sbtqa/tag/stepdefs/ru/MobileSteps.java

```diff
@@ -0,0 +1,32 @@
+package ru.sbtqa.tag.stepdefs.ru;
+
+import cucumber.api.java.Before;
+import cucumber.api.java.en.And;
+import ru.sbtqa.tag.pagefactory.exceptions.SwipeException;
+import ru.sbtqa.tag.stepdefs.MobileGenericSteps;
+
+public class MobileSteps extends MobileGenericSteps {
+
+    @Before(order = 1)
+    public void initMobile() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1161))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=96&issues=AWZh46zKrFHbA8-Egpmy&open=AWZh46zKrFHbA8-Egpmy)

[ISSUE_KEY]: AWZh46zKrFHbA8-Egpmy